src/Configuration/src/SpringBootEnvBase/SpringBootEnvProvider.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.Json;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace Steeltoe.Extensions.Configuration.SpringBootEnv
+{
+    /// <summary>
+    /// Configuration provider that expands spring style '.' delimited configuration keys to .NET compatible format
+    /// </summary>
+    public class SpringBootEnvProvider : ConfigurationProvider
+    {
+        internal ILogger<SpringBootEnvProvider> _logger;
+
+        private const string SPRING_APPLICATION_JSON = "SPRING_APPLICATION_JSON";
+
+        internal string SpringApplicationJson { get; set; }
```

Comment:
should this be kept as a string, or would it make more sense to be a local var inside `Load`?